docs-dev-portal/reading-and-writing-outreach-data.md

```diff
@@ -0,0 +1,109 @@
+# Reading and writing Outreach data
+
+For every extension that your Outreach app contains you configure (in the Outreach Developer Portal) a list of data
+points that your extension can read. To write back data back to Outreach use
+[Outreach REST API](../docs/outreach-api.md).
+
+## The `OutreachContext` object
+
+`extensibilitySdk.init()` call returns a promise that resolves with the `OutreachContext` object. This object carries
+all of the Outreach data available for your extension:
+
+- `locale` - (e.g. `en`) locale currently used in the host application.
+- `theme` - design theme of the host application. Possible values are `Light` or `Dark`.
+- `account` - [account data](#account) configured for your extension.
+- `user` - currently logged-in [user data](#user) configured for your extension.
+- `organization` - [organisation data](#organization) configured for your extension.
+- `opportunity` - [opportunity data](#opportunity) configured for your extension.
+- `prospect` - [prospect data](#prospect) configured for your extension.
+- `config` - user-specific extension [configuration](developer-portal.md#allowing-app-specific-configuration-values) (if
+  any).
+- `host` - info about the host application.
+
+`user`, `organization`, `account`, `opportunity` and `prospect` object contain a subset of the corresponding data
```

Comment:
I don't think this is correct. To be precise I don't think that a reference to Outreach API should be mentioned here.

I had a feeling that these objects only allow access to the context of the entity being displayed. I.e. once on prospect view page and open extensibility tab - that tab iFrame can access that one prospect properties via SDK and not any other prospect or any account/opportunity.

`user` and `organization` are probably special objects that are provided in all contexts

@nimal-outreach please advise